### ISSUE TITLE
Validate OUI theme compliance - phase 1

### ISSUE THREAD

**Issue by joshuarrrr** [opensearch-project/security-dashboards-plugin#1471]
This is a tracking issue for phase 1 of OUI theme plugin compliance, which will involve changes to OUI typography and color SASS variables (for both dark and light mode). **For full details, including dates and tooling, see https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4291**

If your plugin has no UI views or components, please add a comment to that effect, and we'll remove the repo from future look and feel update campaigns.

## Actions required

- [x] Acknowledge the issue by adding an assignee who will serve as a point-of-contact
- [x] Provide screenshots and navigation instructions for important views and screens in your plugin or application. This will assist with both manual and automatic validation of compliance
- [x] Identify all typographic properties and styles
- [x] Create issues to mitigate typographic styles not inherited from OUI components or SASS variables
- [x] Identify all color values, including visualization colors and palettes
- [x] Create issues to mitigate typographic styles not inherited from OUI components or SASS variables
- [x] Resolve all mitigation issues
- [x] Validate and test plugin with updated OUI theme

## Priority issues:

- [x] https://github.com/opensearch-project/security-dashboards-plugin/issues/1524

## Questions?

Add a comment on https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4291

**Comment by scrawfor99** [comment]
[Triage] Hi @joshuarrrr, thank you for making this issue for us. Looking at the linked issue I see:

"We expect to ship the updated theme as an experimental/optional theme in the 2.9 release ([code freeze July 11, 2023](https://opensearch.org/releases.html)), and to make it the default theme in 2.10 ([code freeze August 22, 2023](https://opensearch.org/releases.html)). In order for the UX time to review and approve all plugins, actions should be completed by August 6, 2023." 

Can you confirm when you want these tasks completed? Thank you.

**Comment by joshuarrrr** [comment]
@scrawfor99 Dates may be subject to change, but yes **August 6** is the expected deadline.

**Comment by scrawfor99** [comment]
Hi @joshuarrrr, thanks for the clarification. We just wanted to check whether we should have it all done by 7/11 and then have it approved by 8/6 or just by 8/6.

**Comment by scrawfor99** [comment]
I believe many of these items have been completed by tasks linked off https://github.com/opensearch-project/security-dashboards-plugin/pull/1531. I will handle correcting the issues but it seems like the typography and graphics have already been checked based on the filing of the other issues https://github.com/opensearch-project/security-dashboards-plugin/issues/1529 and #1527.

**Comment by scrawfor99** [comment]
Waiting on this change: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4702

**Comment by scrawfor99** [comment]
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4702 is merged, waiting on backport.

**Comment by scrawfor99** [comment]
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4796 is the backport

**Comment by scrawfor99** [comment]
Hi @joshuarrrr,

I just finished looking things over. 

While validating security dashboards plugin, I noticed that the previous tab on the top bar showing your webpage path seems to auto highlight. See this photo for example:

![Screenshot 2023-08-29 at 3 58 01 PM](https://github.com/opensearch-project/security-dashboards-plugin/assets/65832608/044e0fd7-a2c8-486f-83bb-b1174b6d3f66)

Note that "Dashboards Ma..." is highlighted. This was without my mouse over the button. Based on the behavior in light mode, it looks like this is intentional, but I am not sure if we want the current color choice. 

I also noticed that Security makes use of the grey text instead of the white text that some other areas use:

![Screenshot 2023-08-29 at 4 01 33 PM](https://github.com/opensearch-project/security-dashboards-plugin/assets/65832608/f73d03e3-0ea5-4620-8d30-9160cb8ed46a)

This is not _wrong_, since we still use the theme from core, but I did notice it was different. 

Everything else looks correct.

Let me know if anything else needs to be done, if not I will close this issue.
